lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -48,6 +48,7 @@ class Core
     ["-t", "--timeout"]              => [ true,  "Set a response timeout (default: 15)", "<seconds>"                             ],
     ["-S", "--search"]               => [ true,  "Row search filter.", "<filter>"                                                ],
     ["-x", "--list-extended"]        => [ false, "Show extended information in the session table"                                ],
+    ["-e", "--stale"]                => [ true, "Restrict command to sessions without a recent checkin", "<seconds>"             ],
```

Comment:
Let's get rid of the single flag concept as it might be confusing / break user's intuitions (Without reading the documentation)

```suggestion
    ["--stale"]                => [ true, "Restrict command to sessions without a recent checkin", "<seconds>"             ],
```

Might need to think of a better name for `stale` too 🤔

Reply:
-ci --check-in perhaps?